### ISSUE TITLE
iframe is missing title attribute

### ISSUE THREAD

**Issue by nbaltatzis** [awslabs/amazon-quicksight-embedding-sdk#104]
The iframe used for embedding the dashboard is missing a title attribute. This is an accessibility issue. See: https://www.w3.org/TR/WCAG20-TECHS/H64.html and https://dequeuniversity.com/tips/provide-iframe-titles

Currently, a <title> tag is being added to the <head> section of the of the embedded html document with a value of "Dashboard"

The simple solution for this is just to add a title attribute with value "Dashboard" to the iframe to match the title set for the html document in the head section. 

However, there is a larger issue around this which is that this really needs to be localized correctly based on the `locale` option. I filed a separate issue for that here: https://github.com/awslabs/amazon-quicksight-embedding-sdk/issues/103

Current iframe markup:

```
<iframe class="quicksight-embedding-iframe" width="100%" height="3711" scrolling="no" src="https://us-east-1.quicksight.aws.amazon.com/embed/5ffa...&amp;identityprovider=quicksight&amp;isauthcode=true&amp;undoRedoDisabled=true&amp;resetDisabled=true&amp;punyCodeEmbedOrigin=https://localhost:3000/-&amp;printEnabled=true&amp;locale=es-ES&amp;footerPaddingEnabled=true" style="border: 0px; padding: 0px;"></iframe>
```

**Comment by tulpun** [comment]
Thank you for reporting the issue. We are looking into it.

**Comment by bwelkin** [comment]
`contentOptions.onMessage` in the v2 provide a reference to the iframe in its second argument. Developers can easly set a custom title for the iframe.


```
{
    ...
    onMessage: (messageEvent, metadata) => {
        const {eventName} = messageEvent; 
        const {frame} = metadata; 
        switch (eventName) {
            ...
            case 'CONTENT_LOADED': {
                frame.title = '<YOUR_CUSTOM_IFRAME_TITLE_HERE>';
                break;
            }
            ...
        }
    },
    ...
}
```
